src/middleware/createApiMiddleware.js

```diff
@@ -24,7 +24,7 @@ function handleErrors(response) {
 }
 
 async function handleResponse(response) {
-  const { data, included = [], meta = {} } = await response.json();
+  const { data, included = [], meta = {} } = await response.json()
```

Comment:
Can you re-add the semi-colon here? It'll break the pre-publish linting otherwise.